### ISSUE TITLE
Using <Sight> causes "Uncaught TypeError: Cannot read property 'constructor' of null"

### ISSUE THREAD

**Issue by prescottprue** [React-Sight/React-Sight#84]
When using the provider setup like so:

```jsx
import Signt from 'react-sight'
  ReactDOM.render(
    <Sight>
      <AppContainer store={store} routes={routes} />,
    </Sight>
    document.getElementById('root')
  );
```

I get "Uncaught TypeError: Cannot read property constructor of null"

<img width="1255" alt="screen shot 2017-10-12 at 12 01 57 pm" src="https://user-images.githubusercontent.com/2992224/31514083-5f6c5b0a-af45-11e7-9f70-9544c2149c26.png">

Could it maybe have to do with stateless components not having a constructor? Are stateless components supported?

Did I maybe miss something in setup?

**Comment by davidcsally** [comment]
Thanks for trying it out!

Those console logs are a little worrying to me! It looks like you're using our npm module - this was our beta code but we forgot to take it down. We started out as an npm module but moved all of its functionality to our chrome extension. The module no longer works. We updated npm to point to this repo for instructions.

To use React Sight, you only need to install the chrome extension.
